xml/depl_nodes.xml

```diff
@@ -6847,37 +6846,69 @@ monitoring</guimenu>
-        the LB Network (for example: 172.16.0.0/12), IPv6 subnets can be
-        used for the LB Network. The LB Network is isolated from tenant
-        networks on the amphorae by means of network namespaces on the
-        amphorae. Therefore, operators need not be concerned about
-        overlapping subnet ranges with tenant networks.
-      </para>
-      <para>
-        For more information, see the <link
-        xlink:href="https://docs.openstack.org/octavia/rocky/contributor/guides/dev-quick-start.html#load-balancer-network-configuration">Load
-        Balancer Network Configuration</link> section from the &ostack;
-        documentation.
-      </para>
-      <para>
-        By default the name is <literal>lb-mgmt-net</literal>, this can
-        be overridden in the <literal>raw</literal> mode of the
-        barclamp.
-      </para>
-      <para>
-        The backend services (deployed by the
-        <literal>octavia-backend</literal> role) needs to be able to
-        talk to amphorae over the <literal>lb-mgmt-net</literal>.
-        Connect the <literal>lb-mgmt-net</literal> network to an
-        external network outside the cloud network, or a network that
-        the backend can connect through. Ensure the backend nodes know
-        how to route to it.
+        To configure the &octavia; management network, the network configuration
+        must be initialized or updated to include an <literal>octavia</literal>
+        network entry. The &octavia; &barcl; uses this information to automatically
+        create the neutron provider network used for management traffic.
       </para>
+
+       <procedure>
+        <step>
+         <para>
+          If you have not yet deployed &crow;, add the following configuration to
+          <filename>/etc/crowbar/network.json</filename>
+          to set up the &octavia; management network, using the applicable VLAN
+          ID, and network address values. If you have already deployed &crow;, then add
+          this configuration to the <guimenu>Raw</guimenu> view of the Network &Barcl;.
+         </para>
+         <screen>"octavia": {
+              "conduit": "intf1",
+              "vlan": <replaceable>450</replaceable>,
+              "use_vlan": true,
+              "add_bridge": false,
+              "subnet": "<replaceable>172.31.0.0</replaceable>",
+              "netmask": "<replaceable>255.255.0.0</replaceable>",
+              "broadcast": "<replaceable>172.31.255.255</replaceable>",
+              "ranges": {
+                "host": { "start": "<replaceable>172.31.0.1</replaceable>",
+                   "end": "<replaceable>172.31.0.255</replaceable>" },
+                "dhcp": { "start": "<replaceable>172.31.1.1</replaceable>",
+                   "end": "<replaceable>172.31.255.254</replaceable>" }
+              }
+        },</screen>
+         <para>
+          Care should be taken to ensure the IP subnet doesn't overlap with any of those
```

Comment:
Make sure vlan is unique as well?

Reply:
@guangyee oh, right, that also reminds me I need to warn them about using a VLAN ID that is assigned to the fixed network